percolator/test_procedures/default.json

```diff
@@ -61,107 +62,59 @@
+          "clients": {{ percolator_with_content_hurricane_katrina_search_clients or search_clients | default(1) }}
         },
         {
           "operation": "percolator_with_content_google",
           "warmup-iterations": 100,
-          "iterations": 100
-          {%- if not target_throughput %}
-          ,"target-throughput": 27
-          {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
-          {%- else %}
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%- if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ percolator_with_content_google_target_throughput or target_throughput | default(27) | tojson }},
+          "clients": {{ percolator_with_content_google_search_clients or search_clients | default(1) }}
         },
         {
           "operation": "percolator_no_score_with_content_google",
           "warmup-iterations": 100,
-          "iterations": 100
-          {%- if not target_throughput %}
-          ,"target-throughput": 100
-          {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
-          {%- else %}
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%- if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ percolator_no_score_with_content_google_target_throughput or target_throughput | default(100) | tojson }},
+          "clients": {{ percolator_no_score_with_content_google_search_clients or search_clients | default(1) }}
         },
         {
           "operation": "percolator_with_highlighting",
           "warmup-iterations": 100,
-          "iterations": 100
-          {%- if not target_throughput %}
-          ,"target-throughput": 50
-          {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
-          {%- else %}
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%- if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ percolator_with_highlighting_target_throughput or target_throughput | default(50) | tojson }},
+          "clients": {{ percolator_with_highlighting_search_clients or search_clients | default(1) }}
         },
         {
           "operation": "percolator_with_content_ignore_me",
           "warmup-iterations": 10,
           "iterations": 100,
           "#COMMENT": "Be aware that we specify *target-interval* here! This means we issue one query every 12 seconds",
-          "target-interval": 12
+          "target-interval": 12,
```

Comment:
Should this be an updatable parameter in the README?

Reply:
That's a good point.  This operation is particularly slow and being able to tweak this will be useful.  Will add this in a subsequent commit.